nuget.config

```diff
@@ -3,7 +3,7 @@
   <packageSources>
     <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
-    <add key="ProjectReunion-Dependencies" value="https://pkgs.dev.azure.com/ms/ProjectReunion/_packaging/ProjectReunion-Dependencies/nuget/v3/index.json" />
+    <add key="WindowsAppSDK-Dependencies" value="https://pkgs.dev.azure.com/ms/WindowsAppSDK/_packaging/WindowsAppSDK-Dependencies/nuget/v3/index.json" />
```

Comment:
><add key="WindowsAppSDK-Dependencies" value="https://pkgs.dev.azure.com/ms/WindowsAppSDK/_packaging/WindowsAppSDK-Dependencies/nuget/v3/index.json" / [](http://example.com/codeflow?start=4&length=149)

does this exist?

Reply:
Good catch! No, it doesn't (today). It was one of the few references I'd left but accidentally snagged a few.

Until the new feed's defined this should (continue) to be
```
https://pkgs.dev.azure.com/ms/ProjectReunion/_packaging/ProjectReunion-Dependencies/nuget/v3/index.json
```
(can't change ProjectReunion or ProjectReunion-Dependencies today or the URL doesn't exist)

@BenJKuhn @kythant for later follow up